tests/samplers/test_beam_search.py

```diff
@@ -11,7 +11,7 @@
 #   3. Use the model "huggyllama/llama-7b".
 MAX_TOKENS = [128]
 BEAM_WIDTHS = [4]
-MODELS = ["facebook/opt-125m"]
```

Comment:
I find `facebook/opt-125m` only repeats itself. The output of `TinyLlama/TinyLlama-1.1B-Chat-v1.0` is better, although it is still not very sensible.

```text
[2024-09-21T01:57:27Z] >>>0-th hf output:
--
  | [2024-09-21T01:57:27Z] vLLM is a high-throughput and memory-efficient inference and serving engine for LLMs.
  | [2024-09-21T01:57:27Z]
  | [2024-09-21T01:57:27Z] 2. OpenAI GPT-3: OpenAI GPT-3 is a language model pre-trained on a large corpus of text. It can generate human-like text and has been used in various applications such as chatbots, text generation, and translation.
  | [2024-09-21T01:57:27Z]
  | [2024-09-21T01:57:27Z] 3. GPT-Neo: GPT-Neo is an improved version of GPT-3 that has been trained on a larger corpus of text. It has been used in various applications such as chatbots, text generation, and translation.
  | [2024-09-21T01:57:27Z]
  | [2024-09-21T01:57:27Z] 4. GPT-J: GPT-J is a
  | [2024-09-21T01:57:27Z] >>>0-th vllm output:
  | [2024-09-21T01:57:27Z] vLLM is a high-throughput and memory-efficient inference and serving engine for LLMs.
  | [2024-09-21T01:57:27Z]
  | [2024-09-21T01:57:27Z] 2. OpenAI GPT-3: OpenAI GPT-3 is a language model pre-trained on a large corpus of text. It can generate human-like text and has been used in various applications such as chatbots, text generation, and translation.
  | [2024-09-21T01:57:27Z]
  | [2024-09-21T01:57:27Z] 3. GPT-Neo: GPT-Neo is an improved version of GPT-3 that has been trained on a larger corpus of text. It has been used in various applications such as chatbots, text generation, and translation.
  | [2024-09-21T01:57:27Z]
  | [2024-09-21T01:57:27Z] 4. GPT-J: GPT-J is a
  | [2024-09-21T01:57:27Z] >>>1-th hf output:
  | [2024-09-21T01:57:27Z] vLLM is a high-throughput and memory-efficient inference and serving engine for LLMs.
  | [2024-09-21T01:57:27Z]
  | [2024-09-21T01:57:27Z] 2. OpenAI GPT-3: OpenAI GPT-3 is a language model pre-trained on a large corpus of text. It can generate human-like text and has been used in various applications such as chatbots, text generation, and translation.
  | [2024-09-21T01:57:27Z]
  | [2024-09-21T01:57:27Z] 3. GPT-Neo: GPT-Neo is an improved version of GPT-3 that has been trained on a larger corpus of text. It has been used in various applications such as chatbots, text generation, and translation.
  | [2024-09-21T01:57:27Z]
  | [2024-09-21T01:57:27Z] 4. GPT-J: GPT-J is an
  | [2024-09-21T01:57:27Z] >>>1-th vllm output:
  | [2024-09-21T01:57:27Z] vLLM is a high-throughput and memory-efficient inference and serving engine for LLMs.
  | [2024-09-21T01:57:27Z]
  | [2024-09-21T01:57:27Z] 2. OpenAI GPT-3: OpenAI GPT-3 is a language model pre-trained on a large corpus of text. It can generate human-like text and has been used in various applications such as chatbots, text generation, and translation.
  | [2024-09-21T01:57:27Z]
  | [2024-09-21T01:57:27Z] 3. GPT-Neo: GPT-Neo is an improved version of GPT-3 that has been trained on a larger corpus of text. It has been used in various applications such as chatbots, text generation, and translation.
  | [2024-09-21T01:57:27Z]
  | [2024-09-21T01:57:27Z] 4. GPT-J: GPT-J is an
  | [2024-09-21T01:57:27Z] >>>2-th hf output:
  | [2024-09-21T01:57:27Z] vLLM is a high-throughput and memory-efficient inference and serving engine for LLMs.
  | [2024-09-21T01:57:27Z]
  | [2024-09-21T01:57:27Z] 2. OpenAI GPT-3: OpenAI GPT-3 is a language model pre-trained on a large corpus of text. It can generate human-like text and has been used in various applications, such as chatbots, language translation, and text generation.
  | [2024-09-21T01:57:27Z]
  | [2024-09-21T01:57:27Z] 3. GPT-Neo: GPT-Neo is an improved version of GPT-3 that has been trained on a larger corpus of text. It has been used in various applications, such as chatbots, language translation, and text generation.
  | [2024-09-21T01:57:27Z]
  | [2024-09-21T01:57:27Z] 4. GPT-J: GPT
  | [2024-09-21T01:57:27Z] >>>2-th vllm output:
  | [2024-09-21T01:57:27Z] vLLM is a high-throughput and memory-efficient inference and serving engine for LLMs.
  | [2024-09-21T01:57:27Z]
  | [2024-09-21T01:57:27Z] 2. OpenAI GPT-3: OpenAI GPT-3 is a language model pre-trained on a large corpus of text. It can generate human-like text and has been used in various applications, such as chatbots, language translation, and text generation.
  | [2024-09-21T01:57:27Z]
  | [2024-09-21T01:57:27Z] 3. GPT-Neo: GPT-Neo is an improved version of GPT-3 that has been trained on a larger corpus of text. It has been used in various applications, such as chatbots, language translation, and text generation.
  | [2024-09-21T01:57:27Z]
  | [2024-09-21T01:57:27Z] 4. GPT-J: GPT
  | [2024-09-21T01:57:27Z] >>>3-th hf output:
  | [2024-09-21T01:57:27Z] vLLM is a high-throughput and memory-efficient inference and serving engine for LLMs.
  | [2024-09-21T01:57:27Z]
  | [2024-09-21T01:57:27Z] 2. OpenAI GPT-3: OpenAI GPT-3 is a language model pre-trained on a large corpus of text. It can generate human-like text and has been used in various applications such as chatbots, text generation, and translation.
  | [2024-09-21T01:57:27Z]
  | [2024-09-21T01:57:27Z] 3. GPT-Neo: GPT-Neo is an improved version of GPT-3 that has been trained on a larger corpus of text. It has been used in various applications such as chatbots, text generation, and translation.
  | [2024-09-21T01:57:27Z]
  | [2024-09-21T01:57:27Z] 4. T5: T5 is a language model pre-
  | [2024-09-21T01:57:27Z] >>>3-th vllm output:
  | [2024-09-21T01:57:27Z] vLLM is a high-throughput and memory-efficient inference and serving engine for LLMs.
  | [2024-09-21T01:57:27Z]
  | [2024-09-21T01:57:27Z] 2. OpenAI GPT-3: OpenAI GPT-3 is a language model pre-trained on a large corpus of text. It can generate human-like text and has been used in various applications such as chatbots, text generation, and translation.
  | [2024-09-21T01:57:27Z]
  | [2024-09-21T01:57:27Z] 3. GPT-Neo: GPT-Neo is an improved version of GPT-3 that has been trained on a larger corpus of text. It has been used in various applications such as chatbots, text generation, and translation.
  | [2024-09-21T01:57:27Z]
  | [2024-09-21T01:57:27Z] 4. T5: T5 is a language model pre-


```